### DISCUSSION TITLE
Issues with Fallout 76

### DISCUSSION THREAD

**Discussion by albinostudios-clickmeit** [valinet/ExplorerPatcher#3155] [General]
Running Fallout 76 caused this app to crash 
it would cause fallout to minimize to the desktop 3 times and then give an error that the explorer patcher had encountered an error and would have to close , this happened with both steam and xbox live version of F76

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

Check if you enabled the option register as shell extension, disable it by opening up the EP properties application, go to **File Explorer**, and untick **Register as shell extension**.

Here's why: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

Fallout 76 anticheating feature might be causing this if you have the above enabled.

**Comment by KaotikPaladin** [comment]
I have Steam and Xbox version of Fallout 76 and I use them together without problems with ExplorerPatcher installed
